meson.build

```diff
@@ -0,0 +1,83 @@
+project('euclidean-rhythms.lv2','c',version: '0.1.0-SNAPSHOT',default_options: ['default_library=shared','c_std=c99'])
```

Comment:
Note that default_library=shared is already Meson's default. Some people say explicit is better than implicit, some people say that's just wordy and leads to overly long lines... choose as you will. ;)

Reply:
Thanks for the heads-up. I think I belong to the first group, specially since this is the first time I use meson.